### PR TITLE
Pug ships redesign

### PR DESCRIPTION
**Pull request by beccabunny**
**Content (Artwork / Missions / Jobs)**

## Summary
The current pug ships have aged quite a lot since they have been introduced in 2015, especially when seen from an angle that isn't straight top down. Because of this, I redesigned all the four ships currently in game to better fit with the rest of the aestethic of the game.
arfecta comparison
maboro comparison
enfolta comparison
zibruka comparison

## Artwork Checklist
 - [x] I updated the copyright attributions, or decline to claim copyright of any assets produced or modified
 - [x] I created a PR to the [endless-sky-assets repo](https://github.com/EndlessSkyCommunity/endless-sky-assets) with the necessary image, blend, and texture assets: https://github.com/EndlessSkyCommunity/endless-sky-assets/pull/61
 - [x] I created a PR to the [endless-sky-high-dpi repo](https://github.com/endless-sky/endless-sky-high-dpi) with the `@2x` versions of these art assets: https://github.com/endless-sky/endless-sky-high-dpi/pull/274

### CODE REVIEW

**Review by EjoThims — APPROVED** [review]

**Review by roadrunner56 — APPROVED** [review]

**Review by Amazinite — DISMISSED** [review]
I think the new Arfecta is missing a lot of reddish-orangish highlights that the old one had. The new one is very yellow. On my stream it was also pointed out how the old Arfecta had more white in the center, whereas the new Arfecta cuts out some of that white, making the ship even more yellow.
image

The Zibruka could also perhaps use with a bit more red too it, bringing the gradient closer to the center.

**Review by Saugia — APPROVED** [review]
As per discord, any design elements I've had a comment on have been addressed. I agree with Derpy's notation of less orange on the Arfecta, but otherwise, everything looks great!

**Comment by MasterOfGrey** [comment]
> I agree with Derpy's notation of less orange on the Arfecta
> Edit: I have noticed one thing, though. The "saw teeth"/"feathers" on the Enfolta seem a lot more sharp and numerous than all the other ships- maybe tone them down a little to match?

_Then why is there an "approved" label attached to this review?_ -_-

In other commentary: the additional detail now has the effect of making the Enfolta, which always looked a little weird, to look distinctly _bent_ in the thumbnail, as a result of the back sections rising up. The problem here being that I don't think they _look_ like they're rising up. 
Perhaps if there was some element that was visible only on the thumbnail that provided a visual reference to the in-line axis of the ship's nose, it would be more obvious that they do indeed rise up - and make it easier for the brain to process what it's seeing there?

The other comment I have is that one of the things that has appealed to me about Pug ships traditionally is that they _feel_ organic - it sets them apart as massively alien when they appear. The Zibruka, Enfolta and Maboro models largely manage to retain this, with the following caveats:

- The guns having moulded mounts feels off - whereas before when the guns and their mounts looked "bolted on" it enhanced the organic feel, and that's lost somewhat now.
- The hex patterning is all over the shop. Quite aside from the fact that it's entirely missing from the Zibruka, it's way bigger than it was on the originals and it doesn't appear to conform to the shape of the ship - makes me feel like I'm looking at them through a screen door or something, not like it's a feature of the ship's "skin".
- The cockpit on the Enfolta lacks the curlicue-esque elements that keep the organic vibe for the other two.
- The removal of the only 2 dark spikes on the front section of the Enfolta stands out, when they've been used to such great effect on the Maboro.

Now as for the Arfecta... 

- You've introduced a gorgeous scaly effect on the tail of the Maboro, which feels very natural for a larger ship in this vein, and then missed a golden opportunity to amp it up on the Arfecta by removing the raised rear section entirely.
- The reduction in prominence of the dual cockpit definitely gives an improved sense of scale - but possibly too much so, given the relative size of the Arfecta compared to the Maboro.
- You've got the same spinal hump behind the bridge as you do on the Maboro - with precisely none of the character it has on the Maboro.
- Without reiterating the commentaries on the amount of orange, I feel the overall contrast between sections on the white has largely evaporated, making a prominent structure into something that looks like little more than a rear spoiler on a car.
- Previously there was a certain philosophy to the spikes, or at least a certain visual elegance that implied a strange, but at least consistent thought process to it. Now they look violently chaotic, and busy for the sake of it. At this point I would say half of them could just be removed in favour of contrasting greebles or maybe some of that scaly effect from the Maboro again.

TL:DR This is huge progress, but the stylistic consistency has gone down and I think they would benefit greatly from revisiting them with those sorts of elements in mind.

**Review by Saugia — CHANGES_REQUESTED** [review]
In reference to Grey's "bent comment", the model itself is bent in both the original and new versions. However, due to the stylistic and color changes in the newer one, it does look less intentional and more of an illusion due to the smoother flow of the appearance. I think the main culprit, and at the same time a fix, is this shape here- it being grey on the original is what separated the long body from the curved U.

image

While I do agree with Grey that the gun mounts seem a bit too mechanical for an organic ship, I believe they fit the more detailed look of these ships (especially with the saw-spikes), but that's not to say they still couldn't be less intentional and more molded with the body. Not sure what you want to do there.

+1 on the Enfolta Cockpit lacking the wrapping of a few round appendages around it.

+1 Referencing Grey's comment about the lack of Grey in the front of the Enfolta. See above with the body curve statement- it seems these greys are very important to the Enfolta's design as a whole, and can help it by being carried over.

Same with others, without repeating much- it can use some more color contrast with orange, and more blending of the white than just a hard line stop. Though, this might visually start to fight with the detail lining since it wouldn't match the harsher edges of it anymore (but perhaps it wouldn't look bad anyway). 

While I'm not sure I entirely agree on Grey's comment about the spikes, if you do remove some, be sure to do so uniformly so the back "wings" aren't more bare than the front ones before my suggestions. My favorite spikes are still the 6 on the edges of the first pair of "wings", and the ones along the sides of the body. The ones I had suggested for the rear "wings" do still seem like they're at some harsh angles compared to the rest. 

Lastly, after looking at the design again, I'm kind of missing the flow of the rear "wings" in the first one- is there any way you can mimic it? 

image

The blues in the original are very smooth and curvy, and where I have blues in the new marked, they are good. The reds at the bottom, however, are more jagged and rectilinear, unlike the forward "wings". The greens are where I think things could be edited- the ones on the front "wings" to mimic the inset curves of the original, and the ones on the rear "wings" to have a more curved style like the original.

~~My first review was a bit too quick-glanced, my apologizes~~

**Comment by MasterOfGrey** [comment]
> but that's not to say they still couldn't be less intentional and more molded with the body

To be clear, I wanted the weapon mounts to be *less* molded with the body - making the body itself feel more naturally organic, and the weapons be more "after-market" type deals.

**Comment by Zitchas** [comment]
The new models looks beautiful, but they also look a lot less alien. They make more sense to my human aesthetic, and they have a lot more visible seams and edges and connection points. As a result these look more like what we would get from humans trying to build ships that look like the Pug, as opposed to the creations of a race that so far outstrips or own that we can't even guess at their manufacturing processes. I don't think it is a good change for the Pug.

The original Pug ships looks like Vorlon ships (basically a T3+ elder race from Babylon 5) these new ones look more like the white stars, which were hybrid ships that are mostly human and Mimbari with a few little bits of Vorlon tech. (Decidedly much, much less advanced).

I don't object to these, but I do think it is a bit of a loss for the game.

**Comment by Hurleveur** [comment]
Most of these ships are meant to fight humans and be shown to them. On top of that they use tech that is pretty much of the human's strength level.
Also they left behind all kind of tech for us to understand, and have been observing humans for centuries.
Considering all of this, it would make sense they make it not too hard for us to appreciate these ships and try to understand the tech behind it. The human Pugs could be influenced by being close to/observing humans all the time.

Of course, this kind of logic does not work for the Wanderer's Pugs, who could still use the old designs, that could be said to be to the liking of the Wanderers. Or even be different on purpose to appear god-like to them; whereas they dont want to appear that way to humans.

And then, finally, the arfecta...
Maybe it should be of a third style (with the human pugs having designs humans would like, the wanderer pugs having designs the wanderers would like, and the arfecta just being... Pug.)
We could even keep these 2 different looks for them to show they come from different factions.

**Comment by Zitchas** [comment]
Good idea, and good points. I would like it if the different "levels" of Pug ships had different looks to them. The look in this PR is suitable for the human Pug, maybe the Wanderer.

**Review by GefullteTaubenbrust2 — CHANGES_REQUESTED** [review]
A lot of valid points have already been made here and I'm gonna try not to parrot them, so here is what I think of each ship:

Zibruka:

The detailing is great, but there are a few issues I have with this one. One that should be easy to fix is that I think it has too little grunge and color applied in post and looks a bit dull as a result. There's many changes in the design though that I'm not sure I understand why you made them. Notably the remodel is significantly wider on the front and less sharp, and also seems flatter than the original. Those "appendages" in the middle actually point up significantly in the original, which is somewhat lost here. You are also hiding the engines which seems to me like an odd choice. The design is drastically different as a result of this and I see no reason for why that really is.

Enfolta:

The front part I actually really like with all the detail you added, but again I have a few issues with it. I'll start again with post - the hexagonal tiling effect is nice but hardly shows even at the high resolution you provided, so I doubt it would even be visible in game. It also feels like there is just too much orange in the top down with most of the whole ship orange. 
Also again, you have flattened the ship a whole bunch. On top of those two appendages that again seem like they are more flat with this design, you shrunk that bulge it has in the back down significantly, which I think is probably why others have said the model looks "bent". It just looks off without it.

Maboro:

Looks fantastic for the most part. I noticed that the appendage things in the front are much wider than in the original, which looks slightly odd in the top down. There's also significantly less noticable white areas, which I'm not sure is a good thing. And this is basically nitpicking, but the scales in the back aside from looking gorgeous are unevenly spaced, which makes me unhappy. Anyway, this one is my favorite of the bunch.

Arfecta:

It's much better, but I still don't really enjoy looking at this thing. The fact that you have increased both the amount and pointiness of the greebles and spikes also doesn't really help with that. Your redesign is also much wider at the front, which doesn't look that nice from the top down view. In terms of post processing, it is missing a lot of reds and oranges and the human looking turret ports just feel... odd. 
Now I know people will disagree with this point, but I liked the pointy back of the Arfecta. Yes, it is very exaggerated in the original, but here it just looks stubby, like something is missing.

**Comment by Saugia** [comment]
As this PR has been sitting around for a day under 2 months, I'd like to make an executive suggestion. Becca updated these models *much* better-looking than the original pug ships and has mentioned these changes were requested long after he finished them.  It could also be said that when the PR is opened is when changes should be most important. Regardless, due to the current situation, he feels that he has currently lost the motivation to continue any more edits.

As an artist as well, I feel that if we don't do anything with this PR, we are taking a huge loss with not using these model updates. The current pug ships are now very much outdated from everything else and stand out as a huge sore thumb, as within patch 9.15 an overwhelming amount of ships have gained visual updates. 

I personally don't think that withholding these designs is worth the smaller changes that have been suggested (and I, too, have made suggestions). I think these are amazing remodels and should be merged along with all the other ship updates. Keep in mind: a lot of the current ship visual updates are *not* liked by everyone either, and also could have edits. I don't it's fair to stall these due to that principle. 

I vote that we merged these as they are, and then in the future revisit them once again, rather than leave the very old rubber-like Pug we currently have. (And update the PR before doing so).

**Review by Saugia — APPROVED** [review]

**Comment by GefullteTaubenbrust2** [comment]
> As this PR has been sitting around for a day under 2 months, I'd like to make an executive suggestion. Becca updated these models _much_ better-looking than the original pug ships and has mentioned these changes were requested long after he finished them. Though, it could also be said that when the PR was opened, that's when changes requested are also important. Regardless, due to the current situation, he feels that he has currently lost the motivation to continue any more edits.
> 
> As an artist as well, I feel that if we don't do anything with this PR, we are taking a huge loss with not using these model updates. The current pug ships are now very much outdated from everything else and stand out as a huge sore thumb, as within patch 9.15 an overwhelming amount of ships have gained visual updates.
> 
> I personally don't think that withholding these designs is worth the smaller changes that have been suggested (and I, too, have made suggestions). I think these are amazing remodels and should be merged along with all the other ship updates. Keep in mind: a lot of the current ship visual updates are _not_ liked by everyone either, and also could have edits. I don't it's fair to stall these due to that principle.
> 
> I vote that we merged these as they are, and then in the future revisit them once again, rather than leave the very old rubber-like Pug we currently have and have a huge design inconsistency between the Pug and everyone else. And yes, I know they should have a design inconsistency as the odd aliens they are, but I'm talking about a quality inconsistency.
> 
> (And update the PR branch doing anything).

This is a sentiment where my first instinct is to be very skeptical, but in this case, I agree the ships look better than the originals. So I probably wouldn‘t mind them being merged now, but it isn‘t exactly an ideal solution if I‘m honest.

**Comment by Saugia** [comment]
I suppose I'll leave that up to becca then.

**Review by Saugia — APPROVED** [review]
Everything's looking even better now (it looked good before imo)! It seems most changes have been addressed.

I think the only two things that haven't been changed are the amount of spikes being a lot atm, plus the width of the nose being thinner, but these really aren't detrimental to the design and personally to me, the way these design points are currently, they fit the *newer* design. *Not everything has to be the same as the original.* 

They're looking very alienated and Pug like! Approval from me.

**Review by GefullteTaubenbrust2 — APPROVED** [review]
I think they look much better than they did, though I feel my original points still stand. Either way that basically feels like nitpicking at this point, because they look really great now.

**Review by MasterOfGrey — APPROVED** [review]
My comments on the hex patterning, the gun mounts, and almost everything about the Arfecta stand.

However, while I would definitely like to see those things revisited in the future, the Arfecta stands alone anyway and the other stuff is now at least consistent. All the inconsistent bits on the T1 ships are sorted to my satisfaction, so I would no longer be grumpy about this being merged.

**Comment by Zitchas** [comment]
More orange definitely improved things. I think I'm OK with this now. I'll miss the old ones, though. I like the no-seams, no joints, truly molded from clay look of the old pug ships; but these ones are good if I take the perspective that the Pug, in making T1 ships, tried to make ships that could at least pretend to be manufactured by T1 civilizations...

**Comment by Anarchist2** [comment]
The copyright file on the [high-DPI PR](https://github.com/endless-sky/endless-sky-high-dpi/pull/274) is broken.

**Comment by beccabunny** [comment]
> The copyright file on the [high-DPI PR](https://github.com/endless-sky/endless-sky-high-dpi/pull/274) is broken.

Fixed
